modules/custom/az_paragraphs/css/az_paragraphs_full_width.css

```diff
@@ -4,6 +4,11 @@
 .full-width-background {
     max-width: calc(100vw - var(--scrollbar-width) / 2);
 }
+// See https://github.com/az-digital/az_quickstart/pull/1365
+.region-sidebar-first,
+.region-sidebar-second {
+    z-index: 3;
+}
```

Comment:
Open to suggestions here.

Reply:
In theory, maybe sidebar_second wouldn't need a z-index since it's later in the render order anyway -- but with other z-indexes in place it's probably safe. I haven't tested.

Does this CSS only take effect when a full width paragraph is present?

Reply:
Any time this library is called az_paragraphs.az_paragraphs_full_width

https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/az_paragraphs.libraries.yml#L23-L36

Which is called here: https://github.com/az-digital/az_quickstart/blob/main/modules/custom/az_paragraphs/src/Plugin/paragraphs/Behavior/AZDefaultParagraphsBehavior.php#L154